CONTRIBUTING.md

```diff
@@ -182,14 +182,17 @@ To ensure your changes work as expected, you can run the [automated tests](#auto
 * Runs the `bin/bundle_spec` script before starting and points the UI at the bundled spec.
 * Editor available at http://localhost:9090 once the container is up.
 
-`bin/start`
+`bin/dev`
 * Used to set up a new development environment.
 * Calls both `start_editor` and `start_conjur` to bring up both Conjur and Editor containers.
 
 `bin/start_conjur`
 * Used to start a new local Conjur instance based on the project's `docker-compose`.
 * Both the http & https ports are exposed so requests can be made to Conjur from outside the docker network.
 
+`bin/start_dap`
```

Comment:
Should replace references to `DAP` with `Enterprise` in accordance with the new name change for Conjur's supported edition